packages/core/admin/admin/src/hooks/useFetchMarketplaceProviders/utils/api.js

```diff
@@ -2,8 +2,12 @@ import axios from 'axios';
 
 const MARKETPLACE_API_URL = 'https://market-api.strapi.io';
 
-const fetchMarketplacePlugins = async () => {
-  const { data } = await axios.get(`${MARKETPLACE_API_URL}/providers`);
+const fetchMarketplacePlugins = async ({ sort = 'name:asc' } = null) => {
```

Comment:
I think having two default values may not work ideally here, and name:asc is the default behavior of the API already.

Can you try passing the params object to axios directly instead of destructuring? And making the default value an empty object instead of null